frontend/src/components/BannersWrapper.vue

```diff
@@ -79,4 +83,24 @@ const shouldShowReadonlyBanner = computed(() => {
 const shouldShowExternalUrlBanner = computed(() => {
   return !isDev() && needConfigureExternalUrl.value;
 });
+
+const testRole = computed(() => {
+  return hasWorkspacePermissionV1(
+    "bb.permission.workspace.custom-announcement",
+    currentUserV1.value.userRole
+  );
+});
+
+const testPlan = computed(() => {
+  return currentPlan.value === PlanType.ENTERPRISE;
+});
+
+const shouldShowAnnouncementBanner = computed(() => {
+  return (
+    hasWorkspacePermissionV1(
+      "bb.permission.workspace.custom-announcement",
+      currentUserV1.value.userRole
+    ) && currentPlan.value === PlanType.ENTERPRISE
```

Comment:
Even Bytebase is no longer in Enterprise plan, we should still show the announcement banner. Meanwhile we will show the warning that it's using a feature not available in the Enterprise plan and suggest upgrade. See `BannerUpgradeSubscription` for info.